dali/dafilesrv/dafilesrv.cpp

```diff
@@ -574,6 +578,34 @@ int main(int argc,char **argv)
         lf->setMaxDetail(TopDetail);
         lf->beginLogging();
     }
+
+    Owned<IPropertyTree> env = getHPCCEnvironment();
+    if (env)
+    {
+        IPropertyTree *daFileSrv = env->queryPropTree("Software/DafilesrvProcess");
```

Comment:
I think there always has been in every environment ever created to date, but..
We have talked about having multiple, e.g. on different ports, and the environment xml could had >1 I suppose.

I should use the 'instance name' is passed (which it is in our installation) to ensure I pick up the right one.
I'll change.